acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +82,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
-          step "assert no redacted data in syslog" do
-            #sigh.  gee, thanks for the help, beaker!
-            result = dump_puppet_log(agent)
-            raise if result.length < 3
-            string_io_index = 2
-            run_assertions(assertion_code, result[string_io_index].string)
+        step "assert no redacted data in syslog" do
+          #sigh.  gee, thanks for the help, beaker!
+          syslogfile = case agent['platform']
+                         when /fedora|centos|el|redhat|scientific/ then
+                           '/var/log/messages'
+                         when /ubuntu|debian|cumulus/ then
+                           '/var/log/syslog'
+                         else
+                           nil
+                       end
+          if syslogfile
+            result = agent.exec(Command.new("tail -n 100 #{syslogfile}"),
+                                :acceptable_exit_codes => [0, 1]).stdout.chomp
```

Comment:
TBH, I just copied that over from the ![dump_puppet_log](https://github.com/puppetlabs/beaker/blob/3.1.0/lib/beaker/dsl/helpers/puppet_helpers.rb#L285) method in beaker.  From experimenting with `tail`, the only time I could get it to return a 1 was when the file did not exist.  If you'd rather just go with the `on...` approach and not allow 1 as an acceptable exit code because the syslog file *should* always exist, I could do that instead.  WDYT?